lib/geordi/commands/db_clean.rb

```diff
@@ -0,0 +1,47 @@
+desc 'drop-databases', 'Delete local non-whitelisted databases'
+long_desc <<-LONGDESC
+
+Drop non-whitelisted databases from local installations of MySQL/MariaDB and
+  PostgreSQL. Offers to edit the whitelist.
+LONGDESC
+
+option :postgres_only, :aliases => '-P', :type => :boolean,
+  :desc => 'Only clean Postgres', :default => false
+option :mysql_only, :aliases => '-M', :type => :boolean,
+  :desc => 'Only clean MySQL/MariaDB', :default => false
+option :postgres, :banner => 'STRING',
+  :desc => 'Use Postgres port or socket'
+option :mysql, :banner => 'STRING',
```

Comment:
Please replace `STRING` with `PORT_OR_SOCKET`. Geordi's command help will be more self-explaining this way.